### PR TITLE
customer: expose customer in GraphQL

### PR DESCRIPTION
**Pull request by panzerfahrer**

### CODE REVIEW

File: customer/interfaces/dtocustomer/dto.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package dtocustomer
```
Comment (danielpoe) [inline customer/interfaces/dtocustomer/dto.go:1]: this dto obkects are just used in the graphql package? I think in other places we therefore moved it as subpackage to the graphql folder?

File: customer/interfaces/graphql/schema.graphql [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    additionalAddressLines: [String!]
+    telephone:              String!
+    postCode:               String!
+    city:                   String!
+    firstname:              String!
+    lastname:               String!
+    email:                  String!
+}
+
+extend type Query {
```
Comment (carstendietrich) [inline customer/interfaces/graphql/schema.graphql:40]: maybe worth to add some description in the schema. so we can benefit from the auto generated docs in the graphql playground.
http://spec.graphql.org/June2018/#sec-Descriptions

File: test/integrationtest/projecttest/modules/customer/fake_service.go [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+// GetByIdentity retrieves the authenticated customer by Identity
+func (f FakeService) GetByIdentity(_ context.Context, identity auth.Identity) (domain.Customer, error) {
+	if identity != nil {
+		return getCustomer(identity.Subject()), nil
+	}
+
+	return nil, errors.New("not logged in")
+}
+
+// GetId of the customer
```
Comment (carstendietrich) [inline test/integrationtest/projecttest/modules/customer/fake_service.go:50]: Comment doesn't match changed signature.

File: CHANGELOG.md [hunk 4]
```diff
@@ -13,6 +13,14 @@
   * **Breaking**: renamed the following GraphQL types
     * type `Commerce_Cart_BillingAddressFormData` is now `Commerce_Cart_AddressForm`
     * input `Commerce_BillingAddressFormInput` is now `Commerce_Cart_AddressFormInput`
+    
+**customer**
+* **Breaking**: renamed `GetId` to `GetID` in `domain.Customer` interface*
```
Comment (carstendietrich) [inline CHANGELOG.md:18]: please remove trailing asterisk

File: customer/interfaces/graphql/schema.graphql [hunk 5]
```diff
@@ -0,0 +1,43 @@
+type Commerce_Customer_StatusResult {
+    isLoggedIn: Boolean!
+    userID: String!
+}
+
+type Commerce_CustomerResult {
```
[the remark this answers is not in the document]
Comment (tessig) [reply customer/interfaces/graphql/schema.graphql]: also changed `Commerce_Customer_Status_Result`

**Pull request merged**